weather-app/playground/weather-app.js

```diff
@@ -0,0 +1,101 @@
+const yargs = require('yargs')
+const weather = require('./weather.js')
+
+let searchInfo = {
+    temperatureUnit: 'ºC',
+    notation: 'm',
+    place: ''
+}
+
+yargs.command({
+    command: 'forecast',
+    description: 'Weather forecast summary by city name',
+    builder: {
+        unit: {
+            alias: 'u',
+            describe: 'Temperature unit',
+            demandOption: false,
+            type: 'string'
+        },
+        city: {
+            alias: 'c',
+            describe: 'City name',
+            demandOption: true,
+            type: 'string'
+        }
+    },
+    handler(u) {
+        if (u.unit !== undefined) {
+            const unitValue = u.unit.toUpperCase()
+            switch (unitValue) {
```

Comment:
This is duplicated below. You can create a function and call it from both places.

Reply:
Good catch, I'll do it right now

Reply:
It´s done, please take a look